README.md

```diff
@@ -148,6 +148,7 @@ Alternatively disable signing in each target:
 * Add `CODE_SIGNING_REQUIRED` setting with a value of `NO`
 
 ### Swift Package Manager
+* To enable test schemes: `./scripts/setup_spm_tests.sh`
```

Comment:
I'm afraid I will miss this step all the time in the future 😄

Reply:
Yep, it's annoying, but as far as I can tell worth it because of the product advantages for clients. 😛